immun-tracker/src/components/Login/Login.js

```diff
@@ -4,25 +4,25 @@ import React from "react";
 import { connect } from "react-redux";
 
 //React Router
-import { Link } from 'react-router-dom'
+import { Link } from "react-router-dom";
 
 // import Loader from "react-loader-spinner";
-// import { login } from  '../../actions'
+import { login } from  '../../actions/actions'
```

Comment:
You have single quotes here instead of double-quotes. Look into things like prettier that can auto-format this stuff for you.